src/utils/create-sepsets.ts

```diff
@@ -4,37 +4,28 @@ import {
   any,
   append,
   clone,
-  converge,
   drop,
   equals,
   forEach,
-  includes,
-  length,
-  pipe,
-  prop,
   reduce,
-  subtract,
 } from 'ramda'
 
 import { isNotEmpty } from './fp'
 // this can be removed when ">= 11" and use the NodeJS sort method
 import { sort } from 'timsort'
 
 const forEachIndexed = addIndex<IClique>(forEach)
-const sepSetNodesLength: (sepSet: ISepSet) => number = pipe(prop('sharedNodes'), length)
-
-const getSharedNodes = (cliqueA: IClique, cliqueB: IClique) =>
-  reduce(
-    (acc, nodeId) => {
-      if (includes(nodeId, cliqueB.nodeIds)) {
-        return append(nodeId, acc)
-      }
-
-      return acc
-    },
-    [] as string[],
-    cliqueA.nodeIds,
-  )
+const sepSetCompare: (a: ISepSet, b: ISepSet) => number = (a, b) =>
```

Comment:
IMHO, it would be better to create a sort method rather than the compare method used by the sort method, because this way you can put the entire "sort logic" inside a method. 

Because for me, seems like this way you will have the sort logic in two place:
```js
const compareMethod = (...) => { ... }
const foo = sort(someArray, compareMethod) // I must know that I need to use `sort` here

// vs 
const sortSomething = (...) => { 
  return sort(someArray, compareMethod...)
}
const foo = sortSomething(someArray); // I just know the `someArray` will return sorted
```

Anyways, you don't need to do this change now.